src/__tests__/CurrencyCard.test.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import CurrencyCard from '../components/currencyCard/CurrencyCard';
+
+describe('Currency Card Component', () => {
+  test('renders CryptoMetrics on homepage', () => {
+    render(
+      <BrowserRouter><CurrencyCard /></BrowserRouter>,
+    );
+
+    const price = screen.getByText('Current Price:');
+    expect(price).toBeInTheDocument();
+  });
+
+  test('component match with the snapshot', () => {
+    // Assert
+    const tree = renderer.create(<BrowserRouter><CurrencyCard /></BrowserRouter>).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ]  Your app should have unit tests for pure functions in Redux code as well. This will make your app tested well and avoid it to easily crush. 
![image](https://user-images.githubusercontent.com/43877689/225977422-a747ab55-df7b-42ae-9258-354f498ed65e.png)